packages/config/src/config.js

```diff
@@ -1,35 +1,36 @@
 /* eslint-disable camelcase */
 const path = require('path');
-const proxy = require('@redhat-cloud-services/frontend-components-config-utilities/proxy');
+const proxy = require('@redhat-cloud-services/frontend-components-config-utilities/standalone/proxy');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
-let rewriteLineCounter = 0;
-
 module.exports = ({
-    port,
+    port = 1337,
     publicPath,
     appEntry,
     rootFolder,
     https,
     mode,
     appName,
     useFileHash = true,
-    betaEnv = 'ci',
+    betaEnv,
+    env = 'ci-beta',
     sassPrefix,
-    deployment,
     skipChrome2 = false,
     useProxy,
     localChrome,
     customProxy,
-    proxyVerbose,
     routes,
     routesPath,
-    appUrl,
-    exactUrl,
-    disableFallback,
```

Comment:
I'm happy to test out `useProxy` with a Chrome 1 app if you can link one, but these aren't needed for Chrome 2.